.github/workflows/sql-workbench-test-and-build-workflow.yml

```diff
@@ -24,10 +24,16 @@ jobs:
           ref: ${{ env.OPENSEARCH_VERSION }}
           path: sql/OpenSearch-Dashboards
 
+      - name: Get node and yarn versions
+        id: versions_step
+        run: |
+          echo "::set-output name=node_version::$(node -p "(require('./OpenSearch-Dashboards/package.json').engines.node).match(/[.0-9]+/)[0]")"
+          echo "::set-output name=yarn_version::$(node -p "(require('./OpenSearch-Dashboards/package.json').engines.yarn).match(/[.0-9]+/)[0]")"
+
       - name: Setup Node
```

Comment:
Can you use the Dashboards `.nvmrc` file? Similar to what we do in Dashboards: https://github.com/tmarkley/OpenSearch-Dashboards/blob/c67b31e732995207ae432f24b393c902ec326b96/.github/workflows/build_and_test_workflow.yml#L32-L36

Reply:
sure, updated